shared/src/helpers/task.coffee

```diff
@@ -136,9 +137,19 @@ befores[SPACED_PRACTICE_GROUP] = (task, step, stepIndex, isAvailable) ->
       arguments...
     )
 
+isPersonalized = (task, step, stepIndex) ->
+  _.findWhere(task.steps, {group: PERSONALIZED_GROUP})?.id is step.id
+
+befores[INDIVIDUAL_REVIEW] = (task, step, stepIndex, isAvailable) ->
+  return if isPractice(task)
```

Comment:
based on the card, it sounds like this card should show up once in every homework and reading.  As is, it will only show up once for all assignments.  for it to show up once in every reading for example, it would need to return a `step` object based on the proper conditions, like [here](https://github.com/openstax/tutor-js/pull/1845/files#diff-2fe204e9f263bca1bffc1b0895b5ba14R131) for example